README.md

```diff
@@ -3,8 +3,43 @@
 If you are the kind of person who likes thinking _long-term_ and _high-impact_
 this repo is for you!
 
-**D**o **W**hat **Y**ou **L**ove is _not_ a 3-month stepping-stone project ...
+**D**o **W**hat **Y**ou **L**ove is _not_ a stepping-stone project, we're in it for the long-haul. And after a year and a half, we're making sure that the direction we're heading in is clear to everyone who wants to know.
 
-We _very much_ want/need help on this so if you have _time_
+### Our Goal/Mission
+
+> Empower people to sustainably unlock their potential and pursue their passion through technology.
```

Comment:
Exactly where is this discussion _going_? ❓ 
I feel like this is turning into the comment section on YouTube ... :trollface: 
When the objective is really to give constructive feedback on the Pull Request. 💬 

> We _all_ agree with Ben Horowitz: "Don't Follow Your Passion": https://youtu.be/WRYRBGX4lVM ("_follow your contribution_")

Are we going to change the company name to "***forget following your passion***" 
with the motto: 
> "_get a job you hate, be miserable and silence your sadness by watching trash TV_" 

doesn't quite roll of the tongue ... 

Yes, there are _many_ "***failed entrepreneurs***" who were "passionate" about something but weren't able to turn it into a successful business. That is not the case with dwyl.

We are _very_ good at what we do. Ask HMRC, they _love_ what we are doing. 💰 
In the past year we have paid the salaries _several_ teachers/nurses/firemen through taxes we have paid. If that's not something to _celebrate_ then I don't know what is! 🎉  (_well done all dwylers!_) If we do nothing `else` we can _already_ count ourselves successful because we didn't "_fail_" in our first year(s).

> Everyone working @dwyl loves what they are doing; it's a _prerequisite_.
> All our customers/clients _love_ what we do.

@iteles is _attempting_ to _articulate_ that sense of love and _purpose_.
There are _so_ many other places people can work if the _don't_ want to love what they are doing ... we _want_ to be different/special. 🦄 

We want you @markwilliamfirth to **_love_ what you are doing _every day_**! ❤️  ✅  
We want you to be one of the _successful entrepreneurs_ in the video not one of the cynics who look back on their life and have regrets.
 
We aren't just "following our passion" otherwise I would be "DJ" ... 🎶 🎧 🕺 💃 
And Ines would be running craft/scrap-booking biz/workshop ... 😉 

We are making something _useful_ that solves a problem and generates income.
our _product_ will help other people discover what they are _good_ at _and_ can make a living from ***OR*** just find/make more time in their busy lives to do something they love while working somewhere they aren't especially inspired by.

If you or anyone `else` does not like/agree with dwyl's mission as defined by Inês, please suggest a _credible_ alternative rather than just criticising it. 👍